rdt/transformers/datetime.py

```diff
@@ -73,7 +74,11 @@ def get_output_sdtypes(self):
     def _convert_to_datetime(self, data):
         if data.dtype == 'object':
             try:
-                data = pd.to_datetime(data, format=self.datetime_format)
+                pandas_datetime_format = None
+                if self.datetime_format:
+                    pandas_datetime_format = self.datetime_format.replace('%-', '%')
```

Comment:
I don't think this works the desired way. `%-d` for example is supposed to mean you can omit the first digit of the date if it's not necessary. If someone has data like `2-2-2020`, they might give a format like `%-m-%-d-%Y`. If you replace the `-` then you end up with this as the format instead: `02-02-2020`. If you pass data and a datetime format like I originally said, I don't think this will work. If it parses, does it return data in the original format?

Reply:
The `self.datetime_format` ends up 'formating' the string at the end, this line here is just for pandas to parse it to `datetime`. The reason why I'm replacing the `-` is that `pandas` raises an error (it doesn't support this kind of format to parse it to datetime). Since the `-` is just a `end-formating` type (strftime), then when returning on the revese transform the `-` will be applied to the data:

```python
In [1]: from rdt.transformers.datetime import UnixTimestampEncoder
In [2]: ute = UnixTimestampEncoder(datetime_format='%-m-%-d-%Y')
In [3]: data = pd.DataFrame({'date': ['02-02-2020', '1-10-2010', '05-5-2020']})
In [4]: transformed = ute.fit_transform(data, 'date')
In [5]: ute.reverse_transform(transformed)
Out[5]:
        date
0   2-2-2020
1  1-10-2010
2   5-5-2020
```